src/sidebar/store/modules/annotations.ts

```diff
@@ -567,6 +573,37 @@ const savedAnnotations = createSelector(
   annotations => annotations.filter(ann => isSaved(ann)) as SavedAnnotation[],
 );
 
+/**
+ * Return the list of unique users who authored any annotation, ordered by username.
+ */
+const usersWhoAnnotated = createSelector(
```

Comment:
Using a store selector is fine here. This could also be extracted into a helper in future which is used via `useMemo` within the at-mentions popup when it is actually open.

Reply:
Yep, but in order to keep this store-agnostic, the annotations would need to be provided from the parent component, which seemed less obvious than providing a list of users.